src/registrar/views/domain.py

```diff
@@ -692,6 +704,9 @@ def form_valid(self, form):
         except User.DoesNotExist:
             # no matching user, go make an invitation
             return self._make_invitation(requested_email)
+        else:
+            # if user already exists then just send an email
+            self._send_domain_invitation_email(requested_email, add_success=False)
```

Comment:
(Non-blocking) Shouldn't this be around line 721, with the other success message? I'd imagine we don't want to send an email out if a UserDomainRole fails to create (minus an integrity error)?